system-addon/common/Reducers.jsm

```diff
@@ -6,8 +6,8 @@
 const {actionTypes: at} = Components.utils.import("resource://activity-stream/common/Actions.jsm", {});
 const {Dedupe} = Components.utils.import("resource://activity-stream/common/Dedupe.jsm", {});
 
-const TOP_SITES_DEFAULT_LENGTH = 6;
-const TOP_SITES_SHOWMORE_LENGTH = 12;
+const TOP_SITES_DEFAULT_ROWS = 2;
+const TOP_SITES_MAX_SITES_PER_ROW = 6;
```

Comment:
Seems like we actually want a constants file that is usable by `lib/` and `content-src/`, but.. oh well…? There is `content-src/components/TopSites/TopSitesConstants.js` as well as `content-src/lib/constants.js` but not quite what we need here. @k88hudson any suggestions? I suppose most straightforward would just be a `common/Constants.jsm` with a bunch of `this.FOO = val;` and `this.EXPORTED_SYMBOLS = ["FOO, …];` ?